proto/v1/rollout_service.proto

```diff
@@ -368,28 +368,57 @@ message PlanCheckRun {
   // Format: projects/{project}/sheets/{sheet}
   string sheet = 6;
 
-  string detail = 7;
-  repeated Result results = 8;
+  repeated Result results = 7;
 
   message Result {
-    enum Namespace {
-      NAMESPACE_UNSPECIFIED = 0;
-      BYTEBASE = 1;
-      ADVISOR = 2;
+    oneof result {
+      ResultSuccess success = 1;
+      // Failure if the PlanCheckRun itself failed to run.
+      ResultFailure failure = 2;
     }
-    Namespace namespace = 1;
-    int64 code = 2;
-    enum Status {
-      STATUS_UNSPECIFIED = 0;
-      ERROR = 1;
-      WARNING = 2;
-      SUCCESS = 3;
+
+    message ResultFailure {
```

Comment:
Switch the names of ResultFailure and ResultSuccess?

Reply:
fixed